doc/stepbystep/setting_mtu_on_host.md

```diff
@@ -0,0 +1,31 @@
+# Setting Ethernet packet size on host
+
+When connecting to Ethernet cameras, using larger Ethernet packets, a.k.a jumbo packets, can reduce transmission time and allow for higher FPS.
+On the other hand, when using multi-camera setup maximum size packets can burden the network hardware, such as switches, causing dropped frames.
+
+RealSense Ethernet cameras, such as D555, allow users to set the desired packet size, giving users maximum flexibility in their setup.
+Ideally, expected packet size on host should match packet size sent from the camera. In this guide we will show how to set it.
+
+MTU stands for Maximum Transfer Unit, we may use this to indicate packet size.
```

Comment:
Lets add a note about RealSense default recommended MTU for working with D555 (9000)